ocf_datapipes/load/nwp/gfs.py

```diff
@@ -40,11 +42,29 @@ def open_gfs(zarr_path: Union[Path, str]) -> xr.Dataset:
     Returns:
         Xarray dataset of GFS Forecasts
     """
+
+    _log.info("Loading NWP GFS data")
+
     if "*" in zarr_path:
-        gfs = xr.open_mfdataset(zarr_path, engine="zarr", combine="time", chunks="auto")
+        nwp = xr.open_mfdataset(zarr_path, engine="zarr", combine="time", chunks="auto")
     else:
-        gfs = xr.open_dataset(zarr_path, engine="zarr", mode="r", chunks="auto")
+        nwp = xr.load_dataset(zarr_path, engine="zarr", mode="r", chunks="auto")
+
+    nwp = xr.concat([nwp["dlwrf"], nwp["t"], nwp["u"], nwp["v"], nwp["prate"]], "channel")
```

Comment:
could probably make this more general

Reply:
Yeah, would it be possible to set the variables you want in the config?

Reply:
yea, for this run, just use all of them, and later add them into config - https://github.com/openclimatefix/ocf_datapipes/issues/97